pkg/controller/persistentvolume/controller_base.go

```diff
@@ -388,3 +436,56 @@ func isVolumeBoundToClaim(volume *api.PersistentVolume, claim *api.PersistentVol
 	}
 	return true
 }
+
+// storeObjectUpdate updates given cache with a new object version from Informer
```

Comment:
I think it would be a little clearer to wrap this with `storeClaimUpdate` and `storeVolumeUpdate`